db/controllers/app.controllers.js

```diff
@@ -0,0 +1,6 @@
+const endpointsJson = require(`${__dirname}/../../endpoints.json`)
+
+exports.getApi = (request, response) => {
+    console.log('hello from getApi in controller')
```

Comment:
It's great that you are using console logs whilst writing your code but please remove console logs before pushing your code.